### DISCUSSION TITLE
Week 4 Individual Assignment: CSV Map

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#37] [Assignments]
Submit your csv driven maproom link here.

**Comment by lisa58** [comment]
https://lisa58.github.io/DH151/Week4/index.html

**Comment by justisg** [comment]
https://justisg.github.io/DH151/Week4Lab/index.html

**Comment by alicelu170** [comment]
https://alicelu170.github.io/DH151/Week%204.individual/index.html

**Comment by LynnT2** [comment]
https://lynnt2.github.io/DH151/Week%204/index.html

**Comment by MaryJaneHowland** [comment]
https://maryjanehowland.github.io/DH151/Week%204%20-%20Individual%20Coding%20Assignment/index.html
- I don't know how to reset the image sizes?

**Comment by hanarama** [comment]
https://hanarama.github.io/DH151/Week4/datamap.html

Make sure to click on the markers! :)

**Comment by anncong** [comment]
https://anncong.github.io/DH151/Week4/Maps_Pyramids.html

**Comment by michelle14lee** [comment]
https://michelle14lee.github.io/DH151/Week4/LAart.html

**Comment by eliaranita1** [comment]
https://eliaranita1.github.io/DH151/week%204/index.html

**Comment by natgrace** [comment]
https://natgrace.github.io/DH151/Week4/index.html

**Comment by jomh17** [comment]
https://jomh17.github.io/DH151/WEEK%204/index.html

**Comment by danielliuzy** [comment]
https://danielliu524.github.io/DH151/Week4/index.html

**Comment by ryanorlando** [comment]
https://ryanorlando.github.io/Dh151/Week4/

**Comment by zonaliao** [comment]
https://zonaliao.github.io/DH151/Week4/index.html

**Comment by adrianaromero819** [comment]
https://adrianaromero819.github.io/DH151/Week4/index2.html

**Comment by derekwang99** [comment]
https://derekwang99.github.io/DH151/Week%204%20Individual%20Coding%20Assignment/index.html

**Comment by kimjee8955** [comment]
https://kimjee8955.github.io/DH151/Week4/index.html

**Comment by yjwsb233** [comment]
https://yjwsb233.github.io/DH151/Week4/index.html

**Comment by alexthegreat1** [comment]
https://alexthegreat1.github.io/DH151/Week%204/Individual/index.html

**Comment by lisalou36** [comment]
https://lisalou36.github.io/DH151/week%204.1

**Comment by jmtanenbaum** [comment]
https://jmtanenbaum.github.io/DH151/week4/index.html Note: I wanted to make the popup launch when it zooms to the facility mentioned, but couldn't get it to work. I didn't think anyone would be on slack this late.

**Comment by frida-barragan** [comment]
https://github.com/frida-barragan/DH151/blob/main/Week4_idv/index.html

**Reply by frida-barragan** [reply]
https://frida-barragan.github.io/DH151/Week4_idv/index.html

*I just realized I posted the wrong link!

**Comment by ainsleighdouglas** [comment]
https://ainsleighdouglas.github.io/DH-151/Week4/index.html

**Comment by sebdills** [comment]
https://sebdills.github.io/DH151/Week%204/index.html

**Comment by kaicolorado** [comment]
https://kaicolorado.github.io/DH151/Week%204/index.html

**Comment by JuliaWood1** [comment]
https://juliawood1.github.io/DH151/Week%204/index.html

**Comment by kmcinerny** [comment]
https://kmcinerny.github.io/DH151/Week4/index.html

**Comment by ivanma9** [comment]
https://ivanma9.github.io/DH151/Week4/

Not sure why github pages is not reading the same thing my localhost is, but here is what it looks on localhost.

![Screen Shot 2021-04-25 at 11 57 55 PM](https://user-images.githubusercontent.com/43893062/116041592-3c8e7500-a622-11eb-8aa1-a8a356f3cda3.png)

**Comment by kaisanwatanabe** [comment]
https://kaisanwatanabe.github.io/DH151/week%204/

Site doesn't work because of error "net::ERR_ABORTED 404". I tried to figure it out but didn't really understand what was going on.

**Comment by ec028** [comment]
https://ec028.github.io/DH151/Week4/index.html

**Comment by hoz-map** [comment]
https://hoz-map.github.io/DH151/Week4/index.html

**Comment by alekhyavittalam** [comment]
https://alekhyavittalam.github.io/DH151/AssignmentWeek4/index.html
